workflows/subworkflows/trim_and_reformat_reads.cwl

```diff
@@ -77,15 +83,15 @@ steps:
     run: ../../utils/fastq_to_fasta/fastq_to_fasta.cwl
     in:
       fastq: combine_trimmed/result
-      fastq_count: count
+      fastq_count: count_overlapped_reads/count
     out: [ fasta ]
 
   clean_fasta_headers:
     when: $(inputs.fastq_count != 0)
     run: ../../utils/clean_fasta_headers.cwl
     in:
       sequences: convert_trimmed_reads_to_fasta/fasta
-      fastq_count: count
+      fastq_count: count_overlapped_reads/count
```

Comment:
is fastq count needed as input for any steps other than Trimmomatic?

Reply:
I'm using it in condition: when: $(inputs.fastq_count != 0)

Reply:
ah yes! I see this. Thanks